File: Lab2/app/src/main/java/com/example/firstjavaapp/models/Movie.java
package com.example.firstjavaapp.models;

import androidx.annotation.NonNull;
import java.util.ArrayList;

public class Movie {
    public int id;
    public ExternalId externalId;
    public String type;
    public String name;
    public String alternativeName;
    public String description;
    public String slogan;
    public int year;
    public Poster poster;
    public Backdrop backdrop;
    public Rating rating;
    public Votes votes;
    public Videos videos;
    public Budget budget;
    public Fees fees;
    public Distributors distributors;
    public Premiere premiere;
    public Images images;
    public String status;
    public int movieLength;
    public ArrayList<ProductionCompany> productionCompanies;
    public ArrayList<SpokenLanguage> spokenLanguages;
    public ArrayList<Fact> facts;
    public ArrayList<Genre> genres;
    public ArrayList<Country> countries;
    public ArrayList<Object> seasonsInfo;
    public ArrayList<Person> persons;
    public ArrayList<Object> lists;

    @NonNull
    @Override
    public String toString(){
        return this.name + " " + this.year;
    }
}

File: Lab_2_NEW/app/src/main/java/com/example/lab_2_new/models/Movie.kt
package com.example.lab_2_new.models

import android.media.Rating
import android.provider.MediaStore.Images


class Movie {
    var id = 0
    var externalId: ExternalId? = null
    var type: String? = null
    var name: String? = null
    var alternativeName: String? = null
    var description: String? = null
    var slogan: String? = null
    var year = 0
    var poster: Poster? = null
    var backdrop: Backdrop? = null
    var rating: Rating? = null
    var votes: Votes? = null
    var videos: Videos? = null
    var budget: Budget? = null
    var fees: Fees? = null
    var distributors: Distributors? = null
    var premiere: Premiere? = null
    var images: Images? = null
    var status: String? = null
    var movieLength = 0
    var productionCompanies: ArrayList<ProductionCompany>? = null
    var spokenLanguages: ArrayList<SpokenLanguage>? = null
    var facts: ArrayList<Fact>? = null
    var genres: ArrayList<Genre>? = null
    var countries: ArrayList<Country>? = null
    var seasonsInfo: ArrayList<Any>? = null
    var persons: ArrayList<Person>? = null
    var lists: ArrayList<Any>? = null


    override fun toString(): String {
        return "$name $year"
    }
}